### PR TITLE
add Doxygen

### PR DESCRIPTION
**Pull request by mirkofarano**
In this PR a command line for generating Doxygen documentation has been added (`make doc`).
A short description has been added to the `INSTALL.md` file.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ For debugging, each integration test can be run individually and with various co
 
 ### Doxygen
 
-A webpage base on [Doxygen](http://www.doxygen.nl) is generated by running `make doc` (if doxygen is installed).
+A webpage baseed on [Doxygen](http://www.doxygen.nl) is generated by running `make doc` (if doxygen is installed).
```
Comment (symtor) [inline INSTALL.md]: "based"

**Review by symtor — CHANGES_REQUESTED** [review]
Adding a Doxyfile would be overkill, so as long as this works, it's good

* Fix typo in INSTALL.md
* Food for thought: Can the documentation be autogenerated by the CI in an online, public place?

File: INSTALL.md (hunk 1 above)
Comment (mirkofarano) [reply INSTALL.md]: Already done.

**Comment by mirkofarano** [comment]
> Food for thought: Can the documentation be autogenerated by the CI in an online, public place?

I guess so, but I'll need to check when I have more time.

File: INSTALL.md (hunk 1 above)
Comment (symtor) [reply INSTALL.md]: "baseed" is not the correct spelling, it's going in the right direction, but one 'e' too far

Comment (mirkofarano) [reply INSTALL.md]: I did not see the extra 'e'!

**Review by symtor — APPROVED** [review]

**Pull request merged**